src/Compilers/CSharp/Test/Semantic/Semantics/LookupTests.cs

```diff
@@ -1444,6 +1444,72 @@ private void TestLookupSymbolsNestedNamespaces(List<ISymbol> actual_lookupSymbol
             Assert.Contains(aliasY, actual_lookupSymbols);
         }
 
+        [Fact]
```

Comment:
The issue was with delegate creation: #3651. This test is for method calls which were handled correctly already.